### ISSUE TITLE
Documentation Issue for SteamUserStats.SetStat();

### ISSUE THREAD

**Issue by Jedimaster4559** [Facepunch/Facepunch.Steamworks#430]
[https://wiki.facepunch.com/steamworks/SteamUserStats.SetStat](https://wiki.facepunch.com/steamworks/SteamUserStats.SetStat)

According to this documentation, setting the last parameter to `false` allows for prevent running `SteamUserStats.StoreStats()`, however there is no overload that allows the last argument to be boolean.

**Comment by HyperGamesDev** [comment]
Yeah Im confused as well, sorry for bringing it up even though its closed
